src/linker/Linker/ModuleDefinitionExtensions.cs

```diff
@@ -0,0 +1,14 @@
+using Mono.Cecil;
+
+namespace Mono.Linker
+{
+	public static class ModuleDefinitionExtensions
+	{
+
+		public static bool IsCrossgened (this ModuleDefinition module)
```

Comment:
Why is it needed?

Reply:
This used to be in `AssemblyResolver`, I simply splitted that file to fit `<Type>Extensions.cs`. Looks like this is used for writing back pure IL for crossgen'd assemblies: https://github.com/mono/linker/blob/master/src/linker/Linker.Steps/OutputStep.cs#L117-L118